src/anvil.js

```diff
@@ -31,7 +32,14 @@ module.exports = (mcVersion) => {
     async load (x, z) {
       const data = await this.loadRaw(x, z)
       if (data == null) { return null }
-      return nbtChunkToPrismarineChunk(data)
+      const chunk = nbtChunkToPrismarineChunk(data)
+      for (const _blockEntity of (data.value.block_entities?.value?.value ?? data.value.Level.value.TileEntities?.value?.value) || []) {
+        const blockEntityNbt = { type: 'compound', value: _blockEntity }
+        const { x, y, z } = nbt.simplify(blockEntityNbt)
+        const posKey = `${x},${y},${z}`
+        chunk.blockEntities[posKey] = blockEntityNbt
+      }
+      return chunk
```

Comment:
why are you doing this here as opposed to chunk code?

Reply:
Good question, I will remove it for now